src/components/apps/launch/ResourceRequirements.js

```diff
@@ -372,11 +339,11 @@ const StepResourceRequirementsReview = ({
                         <Table>
                             <TableBody>
                                 <ResourceRequirementsReviewRow
-                                    label={t("minCPUCores")}
-                                    value={min_cpu_cores}
+                                    label={t("cpuCores")}
+                                    value={cpu_cores}
```

Comment:
I could be wrong, but I think that this line might have to be left unchanged as well.

Reply:
Note that the `value` here comes from the JSON returned by the `initAppLaunchValues` formatter. So we can rename this label with the new `cpuCores` label, but in order to change the form field name, both the `initAppLaunchValues` formatter and the submission formatter would need to be updated to convert to and from the API JSON and the JSON used by the form fields.

Since both `max_cpu_cores` and `min_cpu_cores` are still needed to populate the CPU cores selection list values, I think it might be less work to just use `max_cpu_cores` as the form field name (though the display label can be updated):

https://github.com/cyverse-de/sonora/blob/886c95d805801a8c989f29e67db2cf2732d68634/src/components/apps/launch/ResourceRequirements.js#L103-L107